### ISSUE TITLE
Accessibility: Textarea: The ‘Help’ information highlighted in red is not conveyed by the screen reader when text area is accessed.

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3440]
Description: The ‘Help’ information highlighted in red is not conveyed by the screen reader when text area is accessed.

Expected: Ideally, corresponding Help information should be conveyed when UI is accessed

Attached the screenshot:

![image](https://user-images.githubusercontent.com/32538291/94518197-1bf66c00-0247-11eb-9840-111d5a0b4faa.png)

**Comment by kavya-b** [comment]
This has to be handled at the Platform Forms level, and is not related to Textarea.
